CMakeLists.txt

```diff
@@ -29,8 +29,8 @@ if(WIN32)
     set(BUILD_ENGINE_TEST_DEFAULT ON)
     set(BUILD_EDITOR_JNI_DEFAULT  ON)
 else()
-    set(BUILD_ENGINE_TEST_DEFAULT OFF)
-    set(BUILD_EDITOR_JNI_DEFAULT  OFF)
+    set(BUILD_ENGINE_TEST_DEFAULT ON)
+    set(BUILD_EDITOR_JNI_DEFAULT  ON)
```

Comment:
Note that these settings will be changed back to OFF
please use the command listed [here](https://tzuchieh.github.io/photon_v2_build_from_source.html#appendix-a-available-cmake-options) to turn them on